pkg/hs/urbit-king/lib/Urbit/Vere/Ames.hs

```diff
@@ -174,6 +174,10 @@ ames inst who isFake enqueueEv stderr =
                 bindSock socketVar
               Right (bs, addr) -> do
                 logTrace $ displayShow ("(ames) Received packet from ", addr)
+                mLatency <- view (networkConfigL . ncAmesLatency)
+                case mLatency of
+                  Nothing -> pure ()
+                  Just latency -> threadDelay $ latency
```

Comment:
So, because we're reading a from a socket, this threadDelay will also block further reading of packets from the socket, not just enqueuing them for ames. As long as this is what you intend and this is a debugging option that'll never get turned on "for real" in the wild, it's probably ok, but I want to check that you mean this.

Reply:
I agree with @eglaysher here: I would do something like:

Replace this:

`atomically (enqueueEv $ hearEv p a bs)`

With this:

```
async $ do
    threadDelay latency
    atomically (enqueueEv $ hearEv p a bs)
```

On the other hand, this is a kinda dirty hack to test latency, so it probably doesn't matter that much either way.

Reply:
This isn't really important, but a minor style comment since you asked for it:

Instead of this:

```
mLatency <- view (networkConfigL . ncAmesLatency)
case mLatency of
  Nothing -> pure ()
  Just latency -> threadDelay $ latency
```

I would write this:

```
view (networkConfigL . ncAmesLatency) >>= \case
  Nothing      -> pure ()
  Just latency -> threadDelay latency
```

Or even

```
view (networkConfigL . ncAmesLatency) >>= maybe (pure ()) threadDelay
```